domains/lnminh1411.json

```diff
@@ -7,5 +7,8 @@
   },
   "record": {
     "CNAME": "lnminh1411.github.io"
+    "MX": "mx1.improvmx.com" 
+    "MX": "mx2.improvmx.com"
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
You cannot use CNAME with other records, use the A records instead (also you screwed up the MX records)

Reply:
updating for the third time...